app/src/main/java/org/gnucash/android/ui/export/ExportFormFragment.java

```diff
@@ -197,12 +207,37 @@ private void onRadioButtonClicked(View view){
                     mExportWarningTextView.setVisibility(View.GONE);
                 }
 				mExportDateLayout.setVisibility(View.VISIBLE);
+				mExportSeparatorLayout.setVisibility(View.GONE);
 				break;
 
 			case R.id.radio_xml_format:
 				mExportFormat = ExportFormat.XML;
 				mExportWarningTextView.setText(R.string.export_warning_xml);
 				mExportDateLayout.setVisibility(View.GONE);
+				mExportSeparatorLayout.setVisibility(View.GONE);
```

Comment:
The mExportSeparatorLayout does not seem to include the `Separator` label. If you open the app and switch to CSV and then back to XML, the label `Separator` is still visible.